compiler-rt/lib/scudo/standalone/combined.h

```diff
@@ -140,6 +140,11 @@ class Allocator {
   typedef typename QuarantineT::CacheT QuarantineCacheT;
 
   void init() {
+#if !defined(SCUDO_PAGE_SIZE)
```

Comment:
I would suggest removing the `if !defined` because it's something we want to ensure no matter if it's a constant. OTOH, it will catch the mistake `SCUDO_PAGE_SIZE=0`

Reply:
Agreed, so removed.